### PR TITLE
Run Karma with headless Chrome and Firefox

### PR DESCRIPTION
**Pull request by foolip**

### CODE REVIEW

**Comment by foolip** [comment]
I verified it works locally. Will later build on this to test Chrome in Travis as well.

File: config/karma.conf.js [hunk 1]
```diff
@@ -77,7 +77,7 @@ function configurator(config) {
     // available browser launchers:
     // https://npmjs.org/browse/keyword/karma-launcher
     // If run in travisCI, use firefox, otherwise, use Chrome.
```
Comment (Hexcles) [inline config/karma.conf.js]: I didn't notice this before. Why do we use different browsers in different envs? Can we also have Chrome running on Travis?

Comment (foolip) [reply config/karma.conf.js]: I guess it was to make it faster originally. I'll send a PR after this to also run Chrome on Travis.

**Comment by foolip** [comment]
https://codecov.io/github/GoogleChromeLabs/confluence/commit/7419116cd9557352efdc6b7a41fea214c56980ca looks good.

**Pull request merged**